base/data/gm4/tags/blocks/waterloggable.json

```diff
@@ -1,25 +1,25 @@
 {
```

Comment:
This list is outdated; see the following list for the updated version:
https://github.com/Gamemode4Dev/GM4_Datapacks/blob/b38efc0da72a50ff763cf909994303c3a70a6bb5/gm4_orb_of_ankou/data/gm4_orb_of_ankou/tags/blocks/water_loggable.json